samza-kafka/src/main/scala/org/apache/samza/system/kafka/KafkaSystemAdmin.scala

```diff
@@ -542,6 +562,20 @@ class KafkaSystemAdmin(
   }
 
   /**
+    * @inheritdoc
+    *
+    * Delete records of partitions in Kafka up to (and include) the corresponding offsets. This only works with Kafka cluster 0.11 or later.
```

Comment:
Minor: "up to (and **including**) the **provided** offsets. **newline** This only "

Reply:
Sure. It is fixed now.